doc/lsst.ip.diffim/tasks/lsst.ip.diffim.MakeKernelTask.rst

```diff
@@ -11,51 +11,28 @@ MakeKernelTask creates a PSF-matching kernel for two images.
 Description
 ===========
 
-Build a Psf-matching kernel using two input images, either as MaskedImages
-(in which case they need to be astrometrically aligned) or Exposures (in which
-case astrometric alignment will happen by default but may be turned off). This
-requires a list of input Sources which may be provided by the calling Task; if
-not, the Task will perform a coarse source detection and selection for this
-purpose. Sources are vetted for signal-to-noise and masked pixels (in both the
-template and science image), and substamps around each acceptable source are
-extracted and used to create an instance of KernelCandidate. Each
-KernelCandidate is then placed within a `~lsst.afw.math.SpatialCellSet`, which is
-used by an ensemble of `~lsst.afw.math.CandidateVisitor` instances to build the
-Psf-matching kernel. These visitors include, in the order that they are
-called: ``BuildSingleKernelVisitor``, ``KernelSumVisitor``, ``BuildSpatialKernelVisitor``,
-and ``AssessSpatialKernelVisitor``.
-
-Upon initialization, the kernel configuration is defined by
-``self.config.kernel.active``. The task creates an `~lsst.afw.math.Warper` from the
-subConfig ``self.config.kernel.active.warpingConfig``. A schema for the selection
-and measurement of candidate `~lsst.ip.diffim.KernelCandidates` is defined, and
-used to initize subTasks selectDetection (for candidate detection) and
-selectMeasurement(for candidate measurement).
+Build a Psf-matching kernel using two input images, either as MaskedImages (in which case they need to be astrometrically aligned) or Exposures (in which case astrometric alignment will happen by default but may be turned off). 
```

Comment:
I don't believe this Task can handle Exposures that are not astrometrically aligned any more. It was certainly my intention to remove that functionality, so if it is still there we should disable it.

Reply:
I took a look and it seems this functionality is not there anymore. I couldn't find it at least. Changed the docs to reflect this.